### ISSUE TITLE
Randomwalk Sizing

### ISSUE THREAD

**Issue by jasontheillustrator** [CodingTrain/Random-Whistle#30]
Can the randomwalk be zoomed in so that it is more dramatic and obvious?
![Sizing](https://user-images.githubusercontent.com/103963888/213013869-61b0f6d1-59a2-46a5-b324-44aaf72b92a8.jpg)

**Comment by KobeLiesenborgs** [comment]
@jasontheillustrator added some code to allow us to change the default zoom level/change it through the URL, would you like some screenshots to determine what size would be best?

**Comment by shiffman** [comment]
Another idea I had would be to just speed up the walk (multiple cycles per frame) and then over time slow it back down? I can experiment with this later!

**Comment by shiffman** [comment]
This actually works quite well! https://deploy-preview-32--choochoo-whistle.netlify.app/?zoom=8 

(Once we settled on one, we should probably set it as the default but leave the zoom parameter behind the scenes!)

**Comment by jasontheillustrator** [comment]
i really like what @shiffman just did. i like the circular shapes. they look bigger and the speed makes the whole thing much more immediate and obvious. do these randomwalks go for a certain amount of time?

**Comment by KobeLiesenborgs** [comment]
@jasontheillustrator you can test different sizes by changing the "zoom=Number" part of the url! that's what I added to test the sizes!

Currently the walks go infinitely but we could limit them if you think that would be better! 

Another idea might be to have them wrap around the edges?

**Comment by shiffman** [comment]
@KobeLiesenborgs made this I only added the link! Right now it will just go forever, we could add a "pause" and/or "download" button?

**Comment by jasontheillustrator** [comment]
Oh gotcha. I was asking because i let the randomwalk run and it looks like it stopped here:
<img width="874" alt="Screenshot 2023-01-18 at 7 17 24 AM" src="https://user-images.githubusercontent.com/103963888/213210102-1b088d11-abc7-4720-b2e7-d321d4eb73a0.png">

**Comment by KobeLiesenborgs** [comment]
That's because the walks don't wrap around the edges, it's still going but no longer on-screen

**Comment by jasontheillustrator** [comment]
Ok, i understand now. I think wrapping would be a good idea. 

I tested out the zooms. Zoom 10 was good, but if we dont think people will stay on the page too long, zoom 20 would be better.

**Comment by jasontheillustrator** [comment]
I made a few layouts here. There is the horizontal layout which modifies what has currently been done to include an input and button. There's a vertical version that is more mobile friendly. There's a vertical version that includes a download button at the bottom in case someone wants to download the randomwalk as a phone background. Also, for all 3 layouts I adjusted the header graphic so the characters are bigger. I've added the new header graphic below.

![Whistle-Webpage-Mockup-2-Horizontal](https://user-images.githubusercontent.com/103963888/213227187-03aa8e4a-3af0-4e2f-b667-e8084a368394.jpg)
![Whistle-Webpage-Mockup-2-Vertical-Download](https://user-images.githubusercontent.com/103963888/213227195-af3f9be5-a295-42f0-977d-8b00ba23f668.jpg)
![Whistle-Webpage-Mockup-2-Vertical](https://user-images.githubusercontent.com/103963888/213227199-ebe2ce14-08d0-472e-89fc-ce707c569721.jpg)
![Whistle-Webpage-Header-NoBar](https://user-images.githubusercontent.com/103963888/213227221-7d4ccfe8-e2ab-4988-9438-76effa02b25f.png)
